### ISSUE TITLE
Summary blog on Liberty and Azure

### ISSUE THREAD

**Issue by gljrobinson** [OpenLiberty/blogs#2969]
Short summary post of the options we have for Azure

**Comment by git4rk** [comment]
On Azure we have two offerings:
- [IBM WebSphere Liberty and Open Liberty on Azure Kubernetes Service](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/ibm-usa-ny-armonk-hq-6275750-ibmcloud-aiops.20210924-liberty-aks?tab=Overview)
- [IBM WebSphere Liberty and Open Liberty on Azure Red Hat OpenShift](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/ibm-usa-ny-armonk-hq-6275750-ibmcloud-aiops.20210823-liberty-aro?tab=Overview)

**Comment by gljrobinson** [comment]
Hi @git4rk is blog this something you could help us to write up?

**Comment by git4rk** [comment]
# Deploy your cloud-native or monolithic Java applications on Azure AKS and ARO

Cloud-native development is at the heart of modern enterprise application architecture, providing the flexibility and scalability needed to meet today's dynamic demands. 
**WebSphere Liberty** and **Open Liberty** are both lightweight, highly modular, and flexible Java application server provided by IBM that is ideal for modern cloud-native microservices and traditional monolithic applications, but they differ slightly in terms of their scope and licensing. WebSphere Liberty is part of the commercial WebSphere family and is suitable for large enterprises requiring enterprise-grade security, performance, and IBM’s global support services. Open Liberty is an open-source variant that is best for development teams or businesses looking for a lightweight, cloud-native Java runtime without needing enterprise features or support. Developers can deploy WebSphere Liberty and Open Liberty on Kubernetes and OpenShift. 

IBM provides solutions to simplify the deployment and management of cloud-native Java apps on cloud platforms like **Azure Kubernetes Service (AKS)** and **Azure Red Hat OpenShift (ARO)**. Just by providing few configuration settings, in a wizard-driven UI, developers can setup a working AKS and ARO environment with a choice of their application or a sample application running on it.

### [WebSphere Liberty/Open Liberty on Kubernetes](https://ibm.biz/liberty-aks)
- **Azure Kubernetes Service (AKS)** is a managed Kubernetes service by Azure that simplifies the deployment, management, and scaling of containerized applications. It provides the flexibility to manage container infrastructure without worrying about the underlying complexities, making it ideal for Java workloads requiring scalability and flexibility.
- **WebSphere Liberty/Open Liberty** on Kubernetes solution allows developers to run Java microservices and applications in containers, leveraging the Kubernetes orchestration for scaling and reliability.
- The solution allows developers to use an existing AKS cluster or create a new one  based on specifications, like, node size, min and max number of nodes for autoscaling, etc.
- Developers have an option to setup Application Gateway Ingress Controller (AGIC) or use the standard load balancer that comes with AKS.
- The solution automatically deploys the WebSphere Liberty Operator (WLO) or Open Liberty Operator (OLO) depending on user’s entitlement choice. It also allows developers to deploy their application or start with a sample application.

### [WebSphere Liberty/Open Liberty on OpenShift](https://ibm.biz/liberty-aro)
-  **Azure Red Hat OpenShift (ARO)** is a fully managed OpenShift service jointly developed by Red Hat and Azure. It’s a Kubernetes-based platform that provides enhanced capabilities like developer tools, CI/CD pipelines, and a robust security model.
- **WebSphere Liberty/Open Liberty** on OpenShift solution allows developers to use an existing ARO cluster or create a new one based on specifications, like, master and worker node size, min and max number of worker nodes for autoscaling, etc.
- The solution automatically deploys the WebSphere Liberty Operator (WLO) or Open Liberty Operator (OLO) depending on user’s entitlement choice. It also allows developers to deploy their application or start with a sample application.

In summary, **WebSphere Liberty** and **Open Liberty** provide an ideal runtime environment for cloud-native and monolithic applications. IBM solutions, like, **WebSphere Liberty/Open Liberty on Kubernetes** and **WebSphere Liberty/Open Liberty on OpenShift** allows developers to setup AKS and ARO environment and deploy their applications with few clicks. Refer to [What are solutions to run the WebSphere family of products on Azure?](https://learn.microsoft.com/en-us/azure/developer/java/ee/websphere-family) for other IBM solutions provided on Azure.

**Comment by git4rk** [comment]
@GraceJansen Is this what you are looking for: https://github.com/OpenLiberty/blogs/issues/2969#issuecomment-2425115131 ?
